### PR TITLE
gh-111576: Improve documention for tkinter.messagebox

### PR DESCRIPTION
**Pull request by serhiy-storchaka**
* Issue: gh-111576

----
:books: Documentation preview :books:: https://cpython-previews--111578.org.readthedocs.build/

### CODE REVIEW

File: Doc/library/tkinter.messagebox.rst [hunk 1]
```diff
@@ -21,19 +22,169 @@ limited to:
...
+   The following options are supported:
+
+      *command*
+         Specifies the function to invoke when the user closes the dialog.
+         The name of the button clicked by the user to close the dialog is
+         passed as argument.
+         This is only available on macOS.
+
+      *default*
+         Gives the :ref:`symbolic name <messagebox-buttons>` of the default button
```
Comment (terryjreedy) [inline Doc/library/tkinter.messagebox.rst:36]:
```suggestion
         Gives the :ref:`symbolic name <messagebox-buttons>` of the button initially selected (highlighted)
```

File: Doc/library/tkinter.messagebox.rst [hunk 2]
```diff
@@ -21,19 +22,169 @@ limited to:
...
+         Specifies a string to display as the title of the message box.
+         This option is ignored on macOS, where platform guidelines forbid
+         the use of a title on this kind of dialog.
+
+      *type*
+         Arranges for a :ref:`predefined set of buttons <messagebox-types>`
+         to be displayed.
+
+
+   Returns the symbolic name of the selected button.
```
Comment (terryjreedy) [inline Doc/library/tkinter.messagebox.rst]:
```suggestion
   Returns the symbolic name of the selected button, which is the one clicked on or the one highlighted when the Enter key is hit.  Change the button selected with the Tab key or arrow keys.
```
Could have :key: markup, but I forget the details.  I am not sure what the truth is for the second sentence.  Same on all systems?  Others will not know either ;-).

**Review by terryjreedy — CHANGES_REQUESTED** [review]

File: Doc/library/tkinter.messagebox.rst (hunk 1 above)
Comment (serhiy-storchaka) [reply Doc/library/tkinter.messagebox.rst:36]: The Tk documentation uses term "selected" not as a synonym of "highlighted" but as "the one selected by the user". Most of documentation are copied from the Tk manpage with small adaptation, it is better to not introduce inconsistencies. The same terminology can be used in other parts of the Tk documentation (for example the default button is mentioned again three lines below).

File: Doc/library/tkinter.messagebox.rst (hunk 2 above)
Comment (serhiy-storchaka) [reply Doc/library/tkinter.messagebox.rst]: It adds too much details which are not even correct. You can select the button by pressing Space or a shortcut (Alt+the underscored letter). Arrow keys do not change the default button on X Window. Some other keys (e.g Shift-Tab) can change the default button.

Tk uses default behavior on every platform, and it is not a good place to describe it.

**Review by terryjreedy — APPROVED** [review]

**Pull request merged**